packages/server/graphql/mutations/endNewMeeting.ts

```diff
@@ -86,6 +86,7 @@ const clearAgendaItems = async (teamId: string) => {
   return r
     .table('AgendaItem')
     .getAll(teamId, {index: 'teamId'})
+    .filter({pinned: false})
```

Comment:
doing this means the tasks created within the agenda item are also persisted.

repro:
1. add an agenda item "watermelon" & pin it
2. add a task "eat it"
3. finish the meeting
4. start a new meeting & navigate to "watermelon". "eat it" is there.

consider cloning the pinned items and re-creating them on meeting end. maybe include a "pinnedParentId" if you like so we know which agenda item it is a clone of (may be useful for analytics later?)

Reply:
Thanks for the repro steps @mattkrick . I've added a clone function so the tasks no longer persist. 

However, when `AgendaList` initially renders, the cloned items have the id but the content is undefined. On the second render, the content exists. I've handled this in the filter function in the frontend but I'll try to find out why this is happening.